tests/functional/tests.sh

```diff
@@ -32,3 +32,24 @@ T_default-user() {
 	}
 	herokuish-test "test-user" "$(fn-source _test-user)"
 }
+
+T_envfile-parse(){
+    source "$(dirname $BASH_SOURCE)/../../include/buildpack.bash"
+    local envfile_content="
```

Comment:
Wondering if we should use a file fixture or possible a heredoc instead of a variable. Bash sometime works in mysterious ways... 😝